frontend/package.json

```diff
@@ -53,12 +53,14 @@
       "@sentry/nextjs": "7.60.1",
       "@sentry/tracing": "7.53.1",
       "@tanstack/react-query": "4.14.5",
+      "@types/http-proxy": "^1.17.11",
       "@vercel/analytics": "1.0.1",
       "algoliasearch": "4.13.1",
       "cookie": "0.5.0",
       "dayjs": "1.10.5",
       "framer-motion": "10.12.0",
       "graphql": "15.5.3",
+      "http-proxy": "^1.18.1",
```

Comment:
Is it weird that tye types/http-proxy and http-proxy versions don't match?

Reply:
> Is it weird that the types/http-proxy and http-proxy versions don't match

types packages usually don't follow the same version numbers, they're a package on their own

> shouldn't we be using a specific version rather than a caret version

I agree, it makes it easier to know what the exact version is without looking into the lock file

Reply:
I fixed this.